docker-compose.yml

```
@@ -0,0 +1,18 @@
+version: "3.9"
```

I do not see anything specific to `v3.9`, please update it to `v3` for a better compatibility

```suggestion
version: "3"
```

---

addressed